source/extensions/filters/network/kafka/serialization/generator.py

```diff
@@ -4,54 +4,54 @@
 
 
 def generate_main_code(serialization_composite_h_file):
-  """
+    """
   Main code generator.
```

Comment:
this doesnt seem to be properly indented - not a big deal and passes linting but should be properly indented i think